packages/api-elements/test/value-of-test.js

```diff
@@ -24,9 +24,9 @@ describe('valueOf NullElement', () => {
   it('returns null with default and samples', () => {
     const element = new NullElement();
     element.attributes.set('default', new NullElement());
-    element.attributes.set('samples', new ArrayElement(
-      new NullElement()
-    ));
+    element.attributes.set('samples', new ArrayElement([
+      new NullElement(),
+    ]));
```

Comment:
The `ArrayElement` constructor didn't change if I read the changes correctly. How do these test setup changes relate to the functionality change?

Reply:
This was mentioned at the end of the PR description:

> The first commit here is another problem I realised with the existing tests using incorrect samples definition. In API Elements the ArrayElement's content should contain an array, the existing code placed the sample value directly in array element's content which is incorrect form for an ArrayElement. It happens to be that these tests contain the sample values to ensure that the sample values are ignored and the content is not used thus they do affect test status.

This is difference between:

```json
{
  “element”: “array”,
  “content”: { “element”: “string” }
}
```

and:

```json
{
  “element”: “array”,
  “content”: [
    { “element”: “string” }
  ]
}
```

Reply:
> This was mentioned at the end of the PR description

🤦‍♂️ , sorry